tfx/examples/cifar10/cifar10_pipeline_native_keras.py

```diff
@@ -0,0 +1,208 @@
+"""CIFAR10 image classification example using TFX.
+
+This example demonstrates how to do data augmentation, transfer learning,
+and inserting TFLite metadata with TFX.
+The trained model can be pluged into MLKit for object detection.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+
+import os
+from typing import Text
+
+import absl
+import tensorflow_model_analysis as tfma
+
+from tfx.components import Evaluator
+from tfx.components import ExampleValidator
+from tfx.components import ImportExampleGen
+from tfx.components import Pusher
+from tfx.components import ResolverNode
+from tfx.components import SchemaGen
+from tfx.components import StatisticsGen
+from tfx.components import Trainer
+from tfx.components import Transform
+from tfx.components.base import executor_spec
+from tfx.components.trainer.executor import GenericExecutor
+from tfx.dsl.experimental import latest_blessed_model_resolver
+from tfx.orchestration import metadata
+from tfx.orchestration import pipeline
+from tfx.orchestration.beam.beam_dag_runner import BeamDagRunner
+from tfx.proto import pusher_pb2
+from tfx.proto import trainer_pb2
+from tfx.proto import example_gen_pb2
+from tfx.types import Channel
+from tfx.types.standard_artifacts import Model
+from tfx.types.standard_artifacts import ModelBlessing
+from tfx.utils.dsl_utils import external_input
+
+_pipeline_name = 'cifar10_native_keras'
+
+# This example assumes that CIFAR10 train set data is stored in
+# ~/cifar10/data/train, test set data is stored in ~/cifar10/data/test, and
+# the utility function is in ~/cifar10. Feel free to customize as needed.
+_cifar10_root = os.path.join(os.environ['HOME'], 'cifar10')
+_data_root = os.path.join(_cifar10_root, 'data')
+# Python module files to inject customized logic into the TFX components. The
+# Transform and Trainer both require user-defined functions to run successfully.
+_module_file = os.path.join(_cifar10_root, 'cifar10_utils_native_keras.py')
+
+# Path which can be listened to by the model server. Pusher will output the
+# trained model here.
+_serving_model_dir_lite = os.path.join(_cifar10_root, 'serving_model_lite',
+                                       _pipeline_name)
+
+# Directory and data locations.  This example assumes all of the images,
+# example code, and metadata library is relative to $HOME, but you can store
+# these files anywhere on your local filesystem.
+_tfx_root = os.path.join(os.environ['HOME'], 'tfx')
+_pipeline_root = os.path.join(_tfx_root, 'pipelines', _pipeline_name)
+# Sqlite ML-metadata db path.
+_metadata_path = os.path.join(_tfx_root, 'metadata', _pipeline_name,
+                              'metadata.db')
+
+def _create_pipeline(pipeline_name: Text, pipeline_root: Text, data_root: Text,
+                     module_file: Text,
+                     serving_model_dir_lite: Text,
+                     metadata_path: Text,
+                     direct_num_workers: int) -> pipeline.Pipeline:
```

Comment:
could you make sure to align with other examples, e.g., https://github.com/tensorflow/tfx/tree/master/tfx/examples/iris

please use a diff tool to show the diff between those files vs the files under iris thus we know which part is changed, thanks!!